app/src/main/res/layout/include_omnibar_toolbar.xml

```diff
@@ -105,6 +105,17 @@
                             android:padding="4dp"
                             android:src="@drawable/ic_loupe_24dp" />
 
+                        <FrameLayout
+                            android:layout_width="wrap_content"
+                            android:layout_height="wrap_content"
+                            android:layout_gravity="start|center_vertical"
+                            android:visibility="gone"
+                            android:layout_marginStart="17dp"
```

Comment:
these values seem pretty random, what's the reason for it?

Reply:
That specific value is to correctly align the cookie banner animation with the logo inside another Lottie animation. When the cookie animation appears (fade in), we hide (fade out) the logo, so they need to be perfectly aligned so it looks fine.